sdk/lib/etherscan.js

```diff
@@ -63,7 +67,7 @@ class Etherscan {
       json: true
     });
 
-    if (data.status !== "1") {
+    if (data.status !== "1" && data.message !== "No transactions found") {
```

Comment:
This would throw an error if the contract had no internal transactions, which I didn't feel like should be the case